lib/constants.js

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
I do it for arbitrary reasons, but I don't think it's a standard enforced by any linter; I'll remove them ;)